x-pack/plugins/aiops/public/hooks/use_time_filter.ts

```diff
@@ -17,8 +17,8 @@ export const useTimefilter = ({
   timeRangeSelector,
   autoRefreshSelector,
 }: UseTimefilterOptions = {}) => {
-  const { services } = useAiOpsKibana();
-  const { timefilter } = services.data.query.timefilter;
+  const { data } = useAiopsContext();
+  const { timefilter } = data.query.timefilter;
```

Comment:
```suggestion
  const {
    data: {
      query: {
        timefilter: { timefilter },
      },
    },
  } = useAiopsContext();
```

Reply:
Updated in [9f1a33c](https://github.com/elastic/kibana/pull/140005/commits/9f1a33cf5f2c266f448dc64c33afd4798321bf18).